pandas/tests/frame/test_convert_to.py

```diff
@@ -175,7 +172,71 @@ def test_to_records_index_name(self):
     def test_to_records_with_unicode_index(self):
         # GH13172
         # unicode_literals conflict with to_records
-        result = DataFrame([{u'a': u'x', u'b': 'y'}]).set_index(u'a')\
+        result = DataFrame([{u'a': u'x', u'b': 'y'}]).set_index(u'a') \
             .to_records()
         expected = np.rec.array([('x', 'y')], dtype=[('a', 'O'), ('b', 'O')])
         tm.assert_almost_equal(result, expected)
+
+    def test_to_records_with_tz_gmt(self):
+        """
+        This test verifies the bug id #13937
+        """
```

Comment:
Can you add this as a comment instead of docstring? (docstrings hide the test name in the output of nosetests)

Also just the number is fine, like `# GH 13937 `